cli_tools/gce_ovf_import/ovf_utils/ovf_utils.go

```diff
@@ -178,27 +183,48 @@ var ovfOSIDToImporterOSID = map[int16]osInfo{
 	114: {description: "Microsoft Windows 8", importerOSIDs: []string{"windows-8-x86-byol"}, nonDeterministic: true},
 	115: {description: "Microsoft Windows 8 64-bit", importerOSIDs: []string{"windows-8-x64-byol"}, nonDeterministic: true},
 	116: {description: "Microsoft Windows Server 2012 R2", importerOSIDs: []string{"windows-2012r2", "windows-2012r2-byol"}},
-	117: {description: "Microsoft Windows Server 2016", importerOSIDs: []string{"windows-2016", "windows-2016-byol"}},
+	117: {description: "Microsoft Windows Server 2016", importerOSIDs: []string{"windows-2016", "windows-2016-byol", "windows-2019", "windows-2019-byol"}}, //as per https://www.ibm.com/support/knowledgecenter/SSZQFR_2.3.2.0/iwd/pct_byos_win_image.html 2016/2019 share the same ID
 	118: {description: "Microsoft Windows 8.1", importerOSIDs: []string{}},
 	119: {description: "Microsoft Windows 8.1 64-bit", importerOSIDs: []string{}},
 	120: {description: "Microsoft Windows 10", importerOSIDs: []string{"windows-10-x86-byol"}, nonDeterministic: true},
 	121: {description: "Microsoft Windows 10 64-bit", importerOSIDs: []string{"windows-10-x64-byol"}, nonDeterministic: true},
-	//TODO: windows-2019, windows-2019-byol
+}
+
+// GetOSInfoForInspectionResults builds OS info from inspection results
+func GetOSInfoForInspectionResults(ir *pb.InspectionResults) (*OsInfo, int16) {
```

Comment:
You can use a type alias to get a friendly name instead of `int16`. Here's an example:

`type OsID int16`

https://github.com/GoogleCloudPlatform/compute-image-tools/blob/20e0f2d562236ae6d7698dbc007e4cf279b1d38f/proto/go/pb/inspect.pb.go#L134-L133

Reply:
This value is intended to be stored in OVF descriptor which uses int16 right now, so introducing a type alias would require casting back and forth between the alias type and int16.

Reply:
makes sense